extensions/amp-video-docking/0.1/amp-video-docking.js

```diff
@@ -591,7 +595,11 @@ export class VideoDocking {
     }
     const relativeY = this.getSlotRelativeY_();
     const {element} = video;
-    const {top, bottom} = getIntersectionRect(element);
+    const intersectionRect = getIntersectionRect(element);
+    if (!isSizedLayoutRect(intersectionRect)) {
```

Comment:
If the video is not sized, ignore. To prevent mis-cached values from triggering docking/undocking.